core/settings/editor-preferences/model.php

```diff
@@ -185,7 +185,7 @@ protected function register_controls() {
 					'type' => Controls_Manager::SWITCHER,
 					'label_on' => esc_html__( 'Show', 'elementor' ),
 					'label_off' => esc_html__( 'Hide', 'elementor' ),
-					'default' => ChecklistModule::is_preference_switch_on() ? 'yes' : '',
+					'default' => Plugin::$instance->modules_manager->get_modules( 'checklist' )->is_preference_switch_on() ? 'yes' : '',
```

Comment:
Why did we need this change? I actually think the previous version is less risky haha

Reply:
We needed the wordpress adapters inside the `is_preference_switch_on` function and this needed the instance
`public function is_preference_switch_on() : bool {
		if ( $this->should_switch_preferences_off() ) {
			return false;
		}

		$user_preferences = $this->wordpress_adapter->get_user_preferences( self::VISIBILITY_SWITCH_ID );

		return 'yes' === $user_preferences || $this->wordpress_adapter->is_new_installation();
	}`